### ISSUE TITLE
Installation on Python 3.5 fails due to tkinter dependencies in PyMsgBox

### ISSUE THREAD

**Issue by ilykos** [asweigart/pyautogui#88]
> Processing pymsgbox-1.0.3
>     Complete output from command python setup.py egg_info:
>     Traceback (most recent call last):
>       File "<string>", line 1, in <module>
>       File "setup.py", line 6, in <module>
>         version = **import**('pymsgbox').**version**
>       File "**init**.py", line 43, in <module>
>         import tkinter
>     ImportError: No module named 'tkinter'

Since the functionality of PyMsgBox is not critical, I would suggest you remove it.

Anyone who has the same issue can bypass it by removing PyMsgBox from requirements in setup.py like this:

```
setup(
    ...
    install_requires=[
         'PyTweening>=1.0.1', 
         'Pillow', 
         'pyscreeze'
         ---remove---> 'PyMsgBox' <---remove---
     ],
    ...
```

**Comment by dragon788** [comment]
You can remove it or install the `python-tkinter` package. If it isn't used in the code it definitely makes sense to remove.

**Comment by asweigart** [comment]
Fixed in PyMsgBox 1.0.6 https://github.com/asweigart/PyMsgBox/commit/55926b55f46caa969c5ddb87990ebea2737bd66f
